dev/docker/hive/Dockerfile

```diff
@@ -22,7 +22,8 @@ LABEL maintainer="support@datastrato.com"
 
 ARG HADOOP_VERSION
 ARG HIVE_VERSION
-ARG JDBC_DIVER_PACKAGE_NAME
```

Comment:
why remove this?

Reply:
Currently, The `JDBC_DIVER_PACKAGE_NAME` argument unused, So we can delete it.